agnpy/emission_regions.py

```diff
@@ -271,6 +275,68 @@ def P_jet_B(self):
         U_B = np.power(self.B.value, 2) / (8 * np.pi) * u.Unit("erg cm-3")
         return (prefactor * U_B).to("erg s-1")
 
+    @property
+    def gamma_max_confined(self):
+        """Maximum gamma factor of electrons that have their Larmour radius R_L smaller then R_b. In SI units Larmour radius is (see e.g. https://en.wikipedia.org/wiki/Gyroradius#Relativistic_case)
+        .. math::
+           R_L=p_{\perp} / q * B \approx = \gamma m_e * c / q * B 
+
+           so:
+
+        .. math::
+            \gamma = e * B * R_b / m_e *c 
```

Comment:
ok, I will make this change, but note this part of the code (blob is a Blob object created before):
In [17]: blob.R_b=1.e14*u.m; print (f"{blob.gamma_max_larmor:.2e}")                                           
5.87e+09

In [18]: blob.R_b=1.e16*u.cm; print (f"{blob.gamma_max_larmor:.2e}")                                          
5.87e+11

even while you change R_b to cm at the initialization of the class, the value can be changed later on to some other unit. This will probably produce problems also everywhere else in the code. Since as far as I know there are no "private" variables in python, I guess the only solution would be to use everywhere self.R_b.to("cm").value instead, annoying and probably slower, but at least idiot-proof